ProjectSidewalk/SidewalkWebpage
# added source col, removed mobile column from label validation table

Resolves #3164 

Added a new column, source. Logs locations of where users did their validations. Possiblities are GalleryCard, GalleryExpanded, LabelMap, ValidateDesktop, ValidateMobile, AdminLabelMap, AdminLabelSearch, AdminUserDashboard, and AdminContributionPage. Since we now have a way to distinguish between ValidateDesktop and ValidateMobile, removed the is_mobile column from the label validation table. 

##### Before/After screenshots (if applicable)
Before:
![image](https://github.com/ProjectSidewalk/SidewalkWebpage/assets/56641275/2275facd-e9bb-426e-b58b-8f94516dce5b)

After:
![image](https://github.com/ProjectSidewalk/SidewalkWebpage/assets/56641275/d2946091-b772-440b-8e25-73db5f11de98)

##### Testing instructions
1. Go to Project Sidewalk and start testing various forms of validation
2. Go to label validation table and confirm correct sources for each validation type.

##### Things to check before submitting the PR
